huaweicloud/services/acceptance/as/data_source_huaweicloud_as_hook_instances_test.go

```diff
@@ -0,0 +1,127 @@
+package as
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDataSourceHookInstances_basic(t *testing.T) {
```

Comment:
Add a note: 
// Before running this test case, you need to prepare an AS group ID with hook instances.